client/src/commands/instantiateCommand.ts

```diff
@@ -50,43 +48,52 @@ export async function instantiateSmartContract(treeItem?: BlockchainTreeItem): P
             }
         }
 
-        const chosenChannel: IBlockchainQuickPickItem<Array<string>> = await UserInputUtil.showChannelQuickPickBox('Choose a channel to instantiate the smart contract on');
-        if (!chosenChannel) {
-            return;
+        if (!channelName && !peerNames && !chaincodeContainerName) {
```

Comment:
I think you can put this if as part of the else. If you are running from debug then fabric will be started